scripts/build-ffmpeg.py

```diff
@@ -280,94 +270,94 @@
             source_dir="build/generic",
             build_dir="build/generic",
+            lgpl_compatible=False,
         ),
     ]
-    package_groups[2] = [
-        # ffmpeg
-        Package(
-            name="ffmpeg",
-            requires=[
-                "aom",
-                "ass",
-                "bluray",
-                "dav1d",
-                "fontconfig",
-                "freetype",
-                "gmp",
-                "lame",
-                "nasm",
-                "opencore-amr",
-                "openh264",
-                "openjpeg",
-                "opus",
-                "speex",
-                "twolame",
-                "vorbis",
-                "vpx",
-                "x264",
-                "x265",
-                "xml2",
-                "xvid",
-                "xz",
-            ],
-            source_url="https://ffmpeg.org/releases/ffmpeg-6.0.tar.xz",
-            build_arguments=[
-                "--disable-alsa",
-                "--disable-doc",
-                "--disable-libtheora",
-                "--disable-mediafoundation",
-                "--enable-fontconfig",
-                "--enable-gmp",
-                "--enable-gnutls" if use_gnutls else "--disable-gnutls",
-                "--enable-gpl",
-                "--enable-libaom",
-                "--enable-libass",
-                "--enable-libbluray",
-                "--enable-libdav1d",
-                "--enable-libfreetype",
-                "--enable-libmp3lame",
-                "--enable-libopencore-amrnb",
-                "--enable-libopencore-amrwb",
-                "--enable-libopenh264",
-                "--enable-libopenjpeg",
-                "--enable-libopus",
-                "--enable-libspeex",
-                "--enable-libtwolame",
-                "--enable-libvorbis",
-                "--enable-libvpx",
-                "--enable-libx264",
+
+    openh264 = Package(
+        name="openh264",
+        requires=["meson", "nasm", "ninja"],
+        source_filename="openh264-2.2.0.tar.gz",
+        source_url="https://github.com/cisco/openh264/archive/refs/tags/v2.2.0.tar.gz",
+        build_system="meson",
+    )
+
+    ff_build_args = [
+        "--disable-alsa",
+        "--disable-doc",
+        "--disable-libtheora",
+        "--disable-mediafoundation",
+        "--enable-fontconfig",
+        "--enable-gmp",
+        "--enable-gnutls" if use_gnutls else "--disable-gnutls",
+        "--enable-libaom",
+        "--enable-libass",
+        "--enable-libbluray",
+        "--enable-libdav1d",
+        "--enable-libfreetype",
+        "--enable-libmp3lame",
+        "--enable-libopencore-amrnb",
+        "--enable-libopencore-amrwb",
+        "--enable-libopenjpeg",
+        "--enable-libopus",
+        "--enable-libspeex",
+        "--enable-libtwolame",
+        "--enable-libvorbis",
+        "--enable-libvpx",
+        "--enable-libopenh264" if disable_gpl else "--enable-libx264",
```

Comment:
If we're having an if/else on `disable_gpl` later  on, it might make more sense to keep all the GPL-related decisions in one place.